content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -1,22 +1,19 @@
 ---
 reviewers:
 - jessfraz
-title: Inject Information into Pods Using a PodPreset
+title: Inject Information into Pods Using a Pod Preset
 content_template: templates/task
 weight: 60
 ---
 
 {{% capture overview %}}
 
-You can use a `PodPreset` object to inject information like secrets, volume
-mounts, and environment variables etc into pods at creation time.
-This task shows some examples on using the `PodPreset` resource.
+This page shows how to use Pod presets to inject information like secrets, volume mounts, and environment variables into Pods at creation time.
```

Comment:
```suggestion
This page shows how to use PodPreset objects to inject information like Secrets, volume mounts, and environment variables into Pods at creation time.
```
?

Reply:
Using `PodPreset objects` makes it clear that the k8s entities are meant. I think that wording is very good.